dev/release-aux/openssl-announce-pre-release.tmpl

```diff
@@ -0,0 +1,49 @@
+
+   OpenSSL version $release ($LABEL)
```

Comment:
Does it make sense to show the patch number here? We also end up with "alpha" twice. This ends up as:

> OpenSSL version 3.0.0-alpha1 (alpha)

Wouldn't this be better as:

> OpenSSL version 3.0 (alpha1)

Reply:
But we are not dropping the last version field in the new versioning scheme, just the letter. So `OpenSSL version 3.0.0 (alpha1)`?

Reply:
Right - but does the patch number hold any significance here? Throughout the documentation, when referring to the series of versions we refer to 3.0 - never 3.0.0. In much the same way as we say 1.1.1 as shorthand when referring to 1.1.1a, 1.1.1b, 1.1.1c etc. To me having a specific patch number makes no sense here. It's not an alpha release for a specific patch - its an alpha release for the whole 3.0 series.

Reply:
Curious, btw, the label should be just "alpha"...  However, I don't really see why we must separate it out, so I would rather throw away the parenthesis altogether.

Reply:
It would be really confusing if we released the final version as 3.0 and then the patch levels were 3.0.1 and so on. And yes the 3.0 refers to the whole series but a concrete release should not lose and gain the patch-level number. At least it is very uncommon among the open-source projects to do so.

Reply:
I hear you.  I just had a quick look at diverse software version histories, and indeed, those following a semantic version scheme (or semantic version-ish) do present all three numbers at all times, and only use MAJOR.MINOR to represent the series (i.e. 3.0 would be equivalent to 3.0.x).

Reply:
The benefit with a stricter three-number release version is that it makes the script simpler.

Reply:
I've made changes that clarify all this